src/models/visualisation.py

```diff
@@ -0,0 +1,213 @@
+"""
+This module contains visualisation functions for vector, raster and graph data.
+"""
+
+from typing import Optional, List, Tuple, Callable
+
+import folium
+import geopandas as gpd
+import networkx as nx
+import shapely.geometry
+from src.constants import CHERNOBYL_COORDS_WGS84, UTM35N, CEZ_DATA_PATH
+
+
+def create_graph_visualisation(
+    m: folium.Map = None,
+    polygon_gdf: gpd.GeoDataFrame = None,
+    color_column: str = "index",
+    graph: Optional[nx.Graph] = None,
+    name: str = "data",
+    folium_tile_list: Optional[List[str]] = None,
+    location: Tuple[float, float] = CHERNOBYL_COORDS_WGS84,
+    crs: str = UTM35N,
+    add_layer_control: bool = False,
+) -> folium.Map:
+    """
+    This function creates a graph visualisation that puts the polygons in `polygon_gdf`
+    and the `graph` on a folium map. It is intended that the graph was build from
+    `polygon_gdf`, but it is not required. If given `m`, it will be put on this existing
+    folium map.
+
+    Args:
+        m (folium.Map, optional): map to add polygons and graph to. Defaults to None.
```

Comment:
```suggestion
        map (folium.Map, optional): map to add polygons and graph to. Defaults to None.
```

Reply:
I had this originally but `map` is a built-in function (https://docs.python.org/3.8/library/functions.html#map), therefore sticking to the standard folium notation of `m` here. Could call it `fmap` or `folium_map` alternatively?

Reply:
I would favour `folium_map` since there's no real reason to use single letter variable names here.

Reply:
Done!